README.md

```diff
@@ -37,11 +28,35 @@ Building each crate should be as straight forward as `cargo build`, but check ea
 
 This library comes with a testing framework that executes against prerecorded sessions to quickly validate code changes without incurring in Azure costs. You can read more about it in the [Mock testing framework's README](https://github.com/Azure/azure-sdk-for-rust/tree/feature/track2/doc/mock_transport.md).
 
+## Need help?
+
+- For reference documentation visit [docs.rs](https://docs.rs/).
+- For examples, go to the `/examples` sub-directory in any library's project folder.
+  - Ex:`/sdk/identity/azure_identity/examples`
```

Comment:
Bullets shouldn't have a single sub-bullet:

```suggestion
- For examples, go to the `/examples` subdirectory in any library's project folder e.g., `/sdk/identity/azure_identity/examples`
```

"subdirectory" is also one word. Plese check for other instances.